### ISSUE TITLE
Can't connect to mongo db when runnig from docker-compose

### ISSUE THREAD

**Issue by kumo829** [kumo829/to-do_list#2]
Is not possible to connect to the mongo database from the users-service when both services are started from docker-compose. If they're executed from docker or locally there is no problem.

I'm able to access to mongo from RoboT3 using IP.

**Comment by kumo829** [comment]
The connection is set using environment variables on the docker-compose file:

```
      - MONGODB_USERS_HOST=172.20.155.178
      - MONGODB_USERS_PORT=27017
      - MONGODB_USERNAME=<<user>>
      - MONGODB_PASSWORD=<<pass>>
```

**Comment by kumo829** [comment]
Solved along the issue #1
